test/services/dissolution/dissolution.service.test.ts

```diff
@@ -0,0 +1,57 @@
+import { AxiosResponse } from 'axios'
+import { assert } from 'chai'
+import { instance, mock, verify, when } from 'ts-mockito'
+
+import DissolutionRequestMapper from 'app/mappers/dissolution/dissolutionRequest.mapper'
+import { DissolutionCreateResponse } from 'app/models/dto/dissolutionCreateResponse'
+import Optional from 'app/models/optional'
+import DissolutionSession from 'app/models/session/dissolutionSession.model'
+import { DissolutionApiClient } from 'app/services/clients/dissolutionApi.client'
+import { DissolutionService } from 'app/services/dissolution/dissolution.service'
+
+import { generateAxiosResponse } from 'test/fixtures/axios.fixtures'
+import { generateDissolutionSession } from 'test/fixtures/session.fixtures'
+
+describe('DissolutionService', () => {
+  let mapper: DissolutionRequestMapper
+  let client: DissolutionApiClient
+  let service: DissolutionService
+
+  let dissolutionCreateResponse: AxiosResponse<DissolutionCreateResponse>
+  let dissolutionSession: DissolutionSession
+
+  const REFERENCE_NUMBER = '123ABC'
+  const TOKEN = 'some-token'
+  const MAPPED_BODY = {
+    directors: [
+      {name: 'Ashamed Alligator', email: 'ashameda@company.com'},
+      {name: 'Sympathetic Hippopotamus', email: 'sympathetich@company.com'},
+      {name: 'Radical Wombat', email: 'radicalw@company.com', onBehalfName: 'Mysterious Boar'}
+    ]
+  }
```

Comment:
type? Also, all object construction should be in fixtures. Don't we already have one called generateDissolutionCreateRequest() ?